android/build.gradle

```
@@ -47,16 +49,20 @@ repositories {
 }
 
 dependencies {
-    implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
-    implementation "com.android.installreferrer:installreferrer:${safeExtGet('installReferrerVersion', '1.1.2')}"
-    def firebaseIidVersion = safeExtGet('firebaseIidVersion', null)
-    if(firebaseIidVersion){
-        implementation "com.google.firebase:firebase-iid:$firebaseIidVersion"
-    }else{
-        def iidVersion = safeExtGet('googlePlayServicesIidVersion', safeExtGet('googlePlayServicesVersion', '17.0.0'))
-        implementation "com.google.android.gms:play-services-iid:$iidVersion"
-    }
+  implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+  implementation "com.android.installreferrer:installreferrer:${safeExtGet('installReferrerVersion', '1.1.2')}"
+  def firebaseBomVersion = safeExtGet("firebaseBomVersion", null)
+  def firebaseIidVersion = safeExtGet('firebaseIidVersion', null)
+  if (firebaseBomVersion) {
+      implementation platform("com.google.firebase:firebase-bom:${firebaseBomVersion}")
+      implementation "com.google.firebase:firebase-iid"
+  } else if(firebaseIidVersion){
+      implementation "com.google.firebase:firebase-iid:${firebaseIidVersion}"
+  }else{
+      def iidVersion = safeExtGet('googlePlayServicesIidVersion', safeExtGet('googlePlayServicesVersion', '17.0.0'))
+      implementation "com.google.android.gms:play-services-iid:$iidVersion"
+  }
 
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.5.2'
-    testImplementation "org.mockito:mockito-core:3.3.3"
+  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.5.2'
+  testImplementation "org.mockito:mockito-core:3.3.3"
```

```suggestion
  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
  testImplementation "org.mockito:mockito-core:3.6.44"
```

bumping these to current version - same as gradle plugin I'll suggest these in github web ui but then commit so this may move forward